python_gdeploy/actions/snapshot_config.py

```diff
@@ -0,0 +1,21 @@
+from python_gdeploy.wrapper import gdeploy_features as gf
+from python_gdeploy.wrapper.gdeploy_wrapper import cook_gdeploy_config
+from python_gdeploy.wrapper.gdeploy_wrapper import invoke_gdeploy
+
+
+def volume_snapshot_config(volume_name, hostname, action, snapname=""):
```

Comment:
currently only snapshot create,delete and clone are supported. Have added a TODO to support other features